### PR TITLE
Added documentation "find-your-way" to the project folder.

### PR DESCRIPTION
**Pull request by DanieleMaselli**
Awesome you want to make a PR  
please see [https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md](https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md) for further infos.

### CODE REVIEW

File: projects/find-your-way/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+authors: 
+  - "Daniele Maselli"
+authors-url: 
+  - "https://github.com/DanieleMaselli"
+layout: project
+title: Find your way 
+type: project
+---
+
+The current documentation shows the results of the course “gestalten in code” in the winter semester at the university of applied science in Potsdam, supervised by Fabian Morón Zirfas. The course provides an introduction of basic programming in P5.Js and Processing with a focus on gestalt psychology and a workshop on physical computing.
```
Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: P5.js not P5.Js

**Comment by fabianmoronzirfas** [comment]
Also no `:` in folder names. This will create huge errors

File: projects/find-your-way/index.md (hunk 1 above)
Comment (DanieleMaselli) [inline projects/find-your-way/index.md]: I try my best fabian. as usual

Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: 👍

**Comment by DanieleMaselli** [comment]
Hei Fabian! i try to leave out the : in the folder name but it goes automatically every time i upload the folder with the name assets/images....i think it's the slash that cause the problem..can you help me?
i could also write - or _ or just assetsImages

**Comment by DanieleMaselli** [comment]
oo i got it!! 😅

**Comment by DanieleMaselli** [comment]
Hello Fabian, sorry for the delay. I review my writing style and changed the folder. I hope that now it's ok

**Comment by fabianmoronzirfas** [comment]
Hi @DanieleMaselli thanks for the changes. I will review it when I'm back in the office. Around end of the month.

File: projects/find-your-way/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+authors: 
+  - "Daniele Maselli"
+authors-url: 
+  - "https://github.com/DanieleMaselli"
+layout: project
+title: Find your way 
+type: project
+---
+
+The current documentation shows the results of the course “gestalten in code” in the winter semester at the university of applied science in Potsdam, supervised by Fabian Morón Zirfas. The course provides an introduction of basic programming in P5.js and Processing with a focus on gestalt psychology and a workshop on physical computing.
```
Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: it's University of Applied Sciences Potsdam (Germany)

File: projects/find-your-way/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+The first project of the University seminar gestalten in code was titled "analog algorithm". 
+The main task of the project was to provide a set of instructions in a human language emulating algorithms in machines. The instructions had to be in one sentence, simple to understand and could be composed with parameters or constants.
+
+## Algorithm part 1.
+
+
+![](./assets/images/algo-1.png)
+
+
+#### _“Draw round shapes in vertical and horizontal which differ from the diameter. Create lines that merge into different points”._
```
Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: Please remove the italic in the titles. Also they are veeeeeeeeeeerrry long.

File: projects/find-your-way/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+## Algorithm part 2.
+
+
+
+
+![](./assets/images/algo-2.png)
+
+
+#### _“Draw circles of various sizes throughout the sheet distant each other. Trace straight lines which intersect but do not touch the circles._
```
Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: To long to italic.

File: projects/find-your-way/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+While looking on different drawings, it is recognizable that every person gives a different touch and express them self in different ways by just using a circle and a line. The choice of the material was free, which also included objects. 
+
+The interesting thing about the output was set by the different visual tension giving from a circle and a line to create a grid. The problems, instaed, was to cover the entire surface for each algorithm performed.
+
+## Algorithm part 3.
+
+
+![](./assets/images/algo-3.png)
+
+#### _Draw twenty filled circles of different sizes._
```
Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: Do you need so many headlines?

File: projects/find-your-way/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+By noting the individual result of the performers, it was immediately recognizable that it has to be more controlled by the material on which it runs. The choice was made by selecting the most suitable result of all the executed algorithm, which gave an idea for creating more visual depth and a percentage of circles, instead of a defined number of circles.
+
+
+
+## Conclusion
+
+![](./assets/images/algo-5.png)
+
+#### _Draw a circle consisting of 20 to 30 filled circles that do not touch each other._
```
Comment (fabianmoronzirfas) [inline projects/find-your-way/index.md]: To many headlines. The ` ###`  will be transformed into HTML <h1234…> tags. These tend to not break and flow out of the page. Please find another way to highlight that.

File: projects/sol-lewitt/index.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+![](./splash.png)
+
+
+The following project shows the result of the seminar project "deconstructing master artist" at the university of applied science in Potsdam supervised by Fabian Morón Zirfas. The course provides an introduction of basic programming in P5.js and Processing with a focus on gestalt psychology and a workshop on physical computing.
+
+
+![](./assets/images/sol-dma.png)
+
+https://upload.wikimedia.org/wikipedia/en/3/36/Sol_LeWitt.jpg
```
Comment (fabianmoronzirfas) [inline projects/sol-lewitt/index.md]: Please add this as a source link not just the URL

File: projects/sol-lewitt/index.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+In his article for the magazine Artforum in 1967, Sol LeWitt gave a definition of conceptual art: "When an artist uses a conceptual form of art, it means that all the planning is done in advance, all decisions are made beforehand and the execution takes place formally, superficial, shallow ... the purpose of the artist engaged in conceptual art, - to do their job intellectually interesting for the viewer, and thus not affecting his soul. "
+
+LeWitt infrequently created paintings. His most famous works - "Red square, white letters" - refers to 1963. Since the 1970s Levitt began working on an early graphics - screen printing. Then, in the artist's work includes in-depth engraving and later - engraving and other techniques.
+
+## Sol Lewitt: Color Bands 
+![](./assets/images/colorbands.png)
+ 
+
+http://baeditions.com/sol-lewitt-artwork/sol-lewitt-color-bands.htm
```
Comment (fabianmoronzirfas) [inline projects/sol-lewitt/index.md]: Same for this URL source. Do you have the rights to copy and distribute those images?

File: projects/sol-lewitt/index.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+## Analyze: Geometric shapes
+ 
+The first step was made by analyzing the studies behind the geometric shapes in his composition, and with the help of computer programs, it has been recreated two prints of his works to be then analyzed in its details. 
+
+We develop his work without the use of color, this in order to show the purity of the geometric forms and to have a better comprehension what is behind the color. His work method is composed with simple and repeated geometric shapes that changes one to another with different combination. By taking a closure look at the geometric shapes, we realized that the origin of the forms are always coming from the same source and repeat itself throughout the canvas. The space separating each repetition is always measured by the same magnitude until arriving to its ultimate point. In most of his drawings, the separation that occurs between one geometry to another, is closed inside a black line. This in order to separate the concentration of the various forms represented through the viewer's vision.
```
Comment (fabianmoronzirfas) [inline projects/sol-lewitt/index.md]: Please check for right grammar.

File: projects/sol-lewitt/index.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+## Analyze: Geometric shapes
+ 
+The first step was made by analyzing the studies behind the geometric shapes in his composition, and with the help of computer programs, it has been recreated two prints of his works to be then analyzed in its details. 
+
+We develop his work without the use of color, this in order to show the purity of the geometric forms and to have a better comprehension what is behind the color. His work method is composed with simple and repeated geometric shapes that changes one to another with different combination. By taking a closure look at the geometric shapes, we realized that the origin of the forms are always coming from the same source and repeat itself throughout the canvas. The space separating each repetition is always measured by the same magnitude until arriving to its ultimate point. In most of his drawings, the separation that occurs between one geometry to another, is closed inside a black line. This in order to separate the concentration of the various forms represented through the viewer's vision. 
+
+#### Sol LeWitt's art served more as a concept rather than be a valuable object.
```
Comment (fabianmoronzirfas) [inline projects/sol-lewitt/index.md]: To long headlines.

**Comment by DanieleMaselli** [comment]
i will review the requested changes ASAP.

**Comment by DanieleMaselli** [comment]
Hei Fabian! I did the changes and correct the grammar, im not sure if the right for the images is correct...
I think as long as i respect the terms for the images it should be ok

**Comment by fabianmoronzirfas** [comment]
Looks good to me.

**Comment by fabianmoronzirfas** [comment]
@DanieleMaselli @deivanchenko very nice. Your projects are live. Please read trough them to spot any errors. 
- https://interface.fh-potsdam.de/gestalten-in-code/projects/find-your-way/ 
- https://interface.fh-potsdam.de/gestalten-in-code/projects/sol-lewitt/

**Pull request merged**